pkg/plugins/plugin_test.go

```diff
@@ -71,30 +72,23 @@ func TestGet(t *testing.T) {
 
 	t.Run("success", func(t *testing.T) {
 		plugin, err := Get(fruitPlugin, fruitImplements)
-		if err != nil {
-			t.Fatal(err)
-		}
-		if p.Name() != plugin.Name() {
-			t.Errorf("no matching plugin with name %s found", plugin.Name())
-		}
-		if plugin.Client() != nil {
-			t.Error("expected nil Client but found one")
-		}
-		if !plugin.IsV1() {
-			t.Error("Expected true for V1 plugin")
-		}
+		assert.NilError(t, err)
+
+		assert.Check(t, is.Equal(p.Name(), plugin.Name()))
+		assert.Check(t, is.Nil(plugin.Client()))
+		assert.Check(t, plugin.IsV1())
 	})
 
 	// check negative case where plugin fruit doesn't implement banana
 	t.Run("not implemented", func(t *testing.T) {
 		_, err := Get("fruit", "banana")
-		assert.Assert(t, errors.Is(err, ErrNotImplements))
+		assert.ErrorIs(t, err, ErrNotImplements)
```

Comment:
Give that this is the only check in this subtest, it doesn't make a lot of difference, but if we anticipate possible future checks to be added, we could make this a `assert.Check` as well;

```suggestion
		assert.Check(t, is.ErrorIs(err, ErrNotImplements))
```

Reply:
I'm fine with both, just went for the shorter one as it's the only check here 😄 
Changed.